srcpkgs/nqp/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'nqp'
 pkgname=nqp
-version=2018.09
+version=2018.10
 revision=1
 build_style=configure
 make_check_target=test
 configure_script="perl Configure.pl"
 configure_args="--prefix=/usr --backends=moar"
 hostmakedepends="perl"
-makedepends="MoarVM"
-depends="MoarVM"
+makedepends="MoarVM=2018.10"
```

Comment:
Dependencies on makedepends and hostmakedepends are not subject to version constraints, they always use the latest version in srcpkgs/